src/views/Admin/Modules/Records/List.vue

```diff
@@ -8,411 +8,30 @@
       header-bg-variant="white"
```

Comment:
Remove the p-2 on line 2
Remove the :header="title on line 6.

Use title for page title instead. Look at how `<portal to="topbar-title">` is used